### PR TITLE
Documentation newline and sentence restructuring

### PR DESCRIPTION
**Pull request by jpreese**
The figure in the commit was never put on a newline it seems. I also read through the document to make sure everything else made sense.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@jpreese : Thanks for your contribution! The author, @rpetrusha, has been notified to review your proposed change.

File: docs/test/live-unit-testing-start.md [hunk 1]
```diff
@@ -352,7 +353,7 @@ In this section, you'll explore how you can use Live Unit Testing to identify, t
 
 ---
 
-This provides enough information for a preliminary investigation of the bug. Either `TestHasEmbeddedSpaces`, the test routine, made an incorrect assumption, or `HasEmbeddedSpaces` does not correctly recognize all embedded spaces. To diagnose and correct the problem, start with the `StringLibrary.HasEmbeddedSpaces` method:
+This provides enough information for a preliminary investigation of the bug. Either `TestHasEmbeddedSpaces` (the test routine), made an incorrect assumption, or `HasEmbeddedSpaces` does not correctly recognize all embedded spaces. To diagnose and correct the problem, start with the `StringLibrary.HasEmbeddedSpaces` method:
```
Comment (rpetrusha) [inline docs/test/live-unit-testing-start.md]: the comma after "(the test routine)" should be deleted.

File: docs/test/live-unit-testing-start.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ ms.workload:
 ---
 # Get started with Live Unit Testing in Visual Studio
 
-When you enable Live Unit Testing in a Visual Studio solution, Live Unit Testing visually depicts your test coverage and the status of your tests. It also dynamically executes tests whenever you modify your code. It provides immediate notification when changes have broken your code and indicates areas for which additional tests are needed.
+When you enable Live Unit Testing in a Visual Studio solution, Live Unit Testing visually depicts your test coverage and the status of your tests. It also dynamically executes tests whenever you modify your code and immediately notifies you when your changes cause tests to fail.
```
Comment (rpetrusha) [inline docs/test/live-unit-testing-start.md]: It also graphically depicts code coverage and code paths that are not covered by existing tests, which was the intent of the end of the original last sentence. I think that continues to be important to note.

Comment (jpreese) [reply docs/test/live-unit-testing-start.md]: Ok, I'll think of a different way to phrase it. As it is now, it seems kind of tacked on.

File: docs/test/live-unit-testing-start.md (hunk 1 above)
Comment (jpreese) [reply docs/test/live-unit-testing-start.md]: I've pushed this change.

**Comment by PRMerger14** [comment]
@jpreese : Thanks for your contribution! The author, @rpetrusha, has been notified to review your proposed change.

File: docs/test/live-unit-testing-start.md (hunk 2 above)
Comment (rpetrusha) [reply docs/test/live-unit-testing-start.md]: Yes, I agree, @jpreese.

**Comment by rpetrusha** [comment]
#sign-off

**Pull request merged**